agentops/llms/tracker.py

```diff
@@ -7,14 +6,14 @@
 
 from ..log_config import logger
 
-from .cohere import CohereProvider
-from .groq import GroqProvider
-from .litellm import LiteLLMProvider
-from .ollama import OllamaProvider
-from .openai import OpenAiProvider
-from .anthropic import AnthropicProvider
-from .mistral import MistralProvider
-from .ai21 import AI21Provider
+from .providers.cohere import CohereProvider
+from .providers.groq import GroqProvider
+from .providers.litellm import LiteLLMProvider
+from .providers.ollama import OllamaProvider
+from .providers.openai import OpenAiProvider
+from .providers.anthropic import AnthropicProvider
+from .providers.mistral import MistralProvider
+from .providers.ai21 import AI21Provider
```

Comment:
a better way to approach this:

under `providers/__init__.py`:

```
from .cohere import CohereProvider
...

__all__ = [
CohereProvider
...
]
```



Finally, in `agentops/llms/tracker.py`:

`from .providers import CohereProvider` 

clean